A43-sdk-authorization.md

```diff
@@ -0,0 +1,364 @@
+A43: SDK authorization
+
+Author(s): [Ashitha Santhosh](https://github.com/ashithasantosh)
+Approver: 
+Status: Draft {Draft, In Review, Ready for Implementation, Implemented}
+Implemented in: <language, ...>
+Last updated: 2021-06-17
+Discussion at: (filled after thread exists)
+
+##Abstract
+
+gRPC is currently implementing an open-source standard authorization solution
+based on Envoy RBAC. This proposal discusses how this framework would be made 
+available to OSS gRPC users to perform per-RPC authorization checks.
+
+##Background
+
+gRPC authorization support which is currently under implementation, internally 
+implements RBAC Engines based on Envoy RBAC policies. RBAC policy provides 
```

Comment:
Yeah, `RBACK engine` itself is indeed implementation details, so not sure if we want to discuss it here.